src/OneExplorer/OneStorage.ts

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import {assert} from 'chai';
```

Comment:
AKAIK, `chai` is for writing testcases.
Maybe following is what you intended?

```suggestion
import {assert} from 'assert';
```

Reply:
@llFreetimell

> AKAIK, chai is for writing testcases.

I thought chai is more advanced assertion library, supporting BDD also.

https://www.chaijs.com/guide/styles/

You can find that they introduce some assert/BDD apis. 

I searched on stackoverflow people discussing about it. It seems there is not exact guide whether to use chai to replace the assert, but there is no reason to do not so. https://stackoverflow.com/questions/36434068/node-js-assert-library-vs-other-assert-libraries.